addons/event/models/event_registration.py

```diff
@@ -32,7 +32,7 @@ def _get_random_barcode(self):
 
     # event
     event_id = fields.Many2one(
-        'event.event', string='Event', required=True)
+        'event.event', string='Event', default=lambda self: self.env.context.get('active_id'), required=True)
```

Comment:
shouldn't we add a `default_event_id` somewhere instead ?

Reply:
The context was already passed. The issue was unnecessary second call to the `default_get` method.
However, I figure it has been resolved...
Refer to my recent lognote on the task : )